resources/js/components/ui/Table/TQueryTable.tsx

```diff
@@ -106,11 +105,15 @@ export const TQueryTable: Component<TQueryTableProps> = (props) => {
 
   const SORTABLE_COLUMN_TYPES = new Set<ColumnType>(["string", "decimal0", "decimal2", "bool", "date", "datetime"]);
 
-  type CellTemplate = ColumnDefTemplate<CellContext<object, unknown>>;
+  /**
+   * The type of the function used as cell in column definition.
+   *
+   * Note: The function must not return a string directly, it needs to be wrapped in a JSX.Element,
+   * e.g. `<>{someString}</>`. It is not possible to express this
+   */
+  type CellFunc = (context: CellContext<object, unknown>) => JSX.Element | undefined;
```

Comment:
funkcja, która zwraca JSX.Element to komponent
więc ten typ mógłby się nazywać CellComponent

Reply:
Nie jestem całkiem pewien, komponent ma jeszcze tę specyfikę że swój input traktuje jako reaktywny, natomiast funkcja nie. Ale nadal to by była dobra zmiana, gdyby nie to że się spóźniłeś - ta zmiana już weszła do RC, a tutaj wisi nie wiem po co naprawdę, bo powinna być zmergowana automatycznie moim zdaniem skoro jest trywialna. Więc teraz to nie wiem czy mam to poprawiać tutaj, czy w RC, czy olać całkiem. @mblajek powiedz co mądrego skoro taki system wybrałeś.

Reply:
Done.